packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -76,18 +83,24 @@ class DropdownItem extends UIComponent<WithAsProp<DropdownItemProps>> {
       children: false,
       content: false,
     }),
-    accessibilityItemProps: PropTypes.object,
     active: PropTypes.bool,
     content: customPropTypes.itemShorthand,
     checkable: PropTypes.bool,
     checkableIndicator: customPropTypes.itemShorthandWithoutJSX,
+    getItemProps: PropTypes.func,
     header: customPropTypes.itemShorthand,
     image: customPropTypes.itemShorthandWithoutJSX,
+    index: PropTypes.number,
+    item: customPropTypes.itemShorthand,
     onClick: PropTypes.func,
     isFromKeyboard: PropTypes.bool,
     selected: PropTypes.bool,
   }
 
+  shouldComponentUpdate(nextProps: DropdownItemProps) {
+    return !_.isEqual(this.props, nextProps)
```

Comment:
to prevent it being re-rendered when a parent re-renders (in this case `Downshift` as it acts as a render prop component).